src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -2350,10 +2351,10 @@ export class PatientViewPageStore {
     }
 
     @computed get sampleMsiHInfo() {
-        return getSampleClinicalDataMapByThreshold(
+        return getSampleClinicalDataMapByKeywords(
             this.clinicalDataForSamples.result,
-            CLINICAL_ATTRIBUTE_ID_ENUM.MSI_SCORE,
-            MSI_H_THRESHOLD
+            CLINICAL_ATTRIBUTE_ID_ENUM.MSI_TYPE,
+            ['Instable']
```

Comment:
curious if this word is really "Instable" or if it's type?  Unstable?    Either way, maybe make this a constant?

Reply:
@alisman yeah, we use Instable/Stable to describe it https://en.wikipedia.org/wiki/Microsatellite_instability. I will update to constant.